### DISCUSSION TITLE
[Projects Beta] Sort by Kanban order at List view?

### DISCUSSION THREAD

**Discussion by yousan** [community/community#9233] [Projects and Issues]
Hi.
We use Projects beta and really good for us.
We used old Projects feature (that is also good) as Kanban board.
Each card are sorted by priority at a Column.

We can switch the view, List or Kanban, but I don't know how to sort by Kanban order at List view.

Can we sort items at List view?

**Comment by chreck** [comment]
It is possible to sort in kanban board by a field ascending or descending, but it does not update or work yet it seems.
